packages/form-js-viewer/src/render/hooks/useOptionsAsync.js

```diff
@@ -32,44 +32,38 @@ export function useOptionsAsync(field) {
     values: staticOptions
   } = field;
 
-  const [ optionsGetter, setOptionsGetter ] = useState({ options: [], error: undefined, loadState: LOAD_STATES.LOADING });
   const initialData = useService('form')._getState().initialData;
-
   const expressionEvaluation = useExpressionEvaluation(optionsExpression);
-  const evaluatedOptions = useDeepCompareState(expressionEvaluation || [], []);
-
-  useEffect(() => {
+  const evaluatedOptions = useDeepCompareMemoize(expressionEvaluation || []);
 
+  const optionsGetter = useMemo(() => {
     let options = [];
 
     // dynamic options
     if (optionsKey !== undefined) {
-      const keyedOptions = (initialData || {})[ optionsKey ];
-
+      const keyedOptions = (initialData || {})[optionsKey];
       if (keyedOptions && Array.isArray(keyedOptions)) {
         options = keyedOptions;
       }
+    }
 
     // static options
-    } else if (staticOptions !== undefined) {
+    else if (staticOptions !== undefined) {
```

Comment:
better move the brace back next to the else :sweat_smile:

Reply:
I'll revert this one because it's historical but maybe something worth adding to style rules. It's inconsistent in the code base.

Personally, I much prefer the braces around each block, I find it reads more natural.

Reply:
We could ask the bpmn.io folks about this one and get a vote going.